### DISCUSSION TITLE
I would like to ask you a question about your project

### DISCUSSION THREAD

**Discussion by Chinthareddy21** [community/community#50384] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Respected sir, 

I'm using your project model for my test and I got a set back. Can you pls help me to fix a page as default like beforeAll in your framework.

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub. In that case you should look for support information *in the repository for that software*, usually in the README or a file called SUPPORT. If those don't say anything, opening an issue there is usually good. :slightly_smiling_face:
